### PR TITLE
Improved en_GB and ro_RO 

### PR DESCRIPTION
**Pull request by the-noob**
en_GB
- improved postcode generator although not guaranteed to produce valid codes
- landline formatter for phone numbers

ro_RO
- adding real text

### CODE REVIEW

File: src/Faker/Provider/en_GB/Address.php [hunk 1]
```diff
@@ -86,4 +86,25 @@ public static function county()
...
+    /**
+     * Attempt to generate something that looks like valid postcode
+     *
+     * @link http://stackoverflow.com/questions/164979/uk-postcode-regex-comprehensive
+     * @example 'N6 5AA'
+     */
+    public static function postcode()
+    {
+        $attempts = 10;
+        do {
```
Comment (fzaninotto) [inline src/Faker/Provider/en_GB/Address.php]: I don't like the iterative approach. You should fix the format instead.

File: src/Faker/Provider/ro_RO/Text.php [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+class Text extends \Faker\Provider\Text
+{
+    /**
+     * Title: Neghiniţă
+     *
+     * Author: Barbu Ştefănescu Delavrancea
+     *
+     * Language: Romanian
+     *
+     * @see http://www.povesti-pentru-copii.com/barbu-stefanescu-delavrancea/neghinita.html
```
Comment (fzaninotto) [inline src/Faker/Provider/ro_RO/Text.php]: The site mentions a copyright, and we need a public license text. Also, please make a separate PR for that.

**Comment by the-noob** [comment]
I'll implement the suggestions and split the changes into 2 separate PRs
